z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -80,58 +81,108 @@ Depending on the type of image registry backend an additional 100 GB volume.
+
+## Install required packages
 
 **Requirements:**
 
 * Python
 * Ansible
+* jq
 * Python modules required in the playbooks. Namely:
   * openstackclient
   * openstacksdk
   * netaddr
 
-### RHEL
+**Verify the installation:**
+```sh
+$ openstack
+(openstack)
+```
+
+**Make sure that `python` points to Python3:**
+```sh
+sudo alternatives --set python /usr/bin/python3
+```
+
+## Generate an SSH private key and add it to the CIC control node
+
+You can use this key to SSH into the ICIC nodes and Bastion Node. 
+1. Create the SSH Key
+```sh
+$ ssh-keygen -f ~/.ssh/id_rsa -t rsa
+```
+2. Add your SSH key to ICIC control nodes and bastion nodes.
+```sh
+$ ssh-copy-id root@<CIC-control-node-ip>
+$ ssh-copy-id root@<Bastion-Node>
+```
 
-From a RHEL 8 box, make sure that the repository origins are all set:
+## Connect the CIC client by using command line
 
+1. Copy the `icicrc` file to your user's home directory.
 ```sh
-sudo subscription-manager register # if not done already
-sudo subscription-manager attach --pool=$YOUR_POOLID # if not done already
-sudo subscription-manager repos --disable=* # if not done already
+$ mkdir -p opt/ibm/icic/icicrc
+$ scp -r root@<CIC-control-node-ip>:/opt/ibm/icic/icicrc opt/ibm/icic/icicrc
+```
 
-## TODO replace repos to s390x repos
-sudo subscription-manager repos \
-  --enable=rhel-8-for-s390x-baseos-rpms \
-  --enable=openstack-16-tools-for-rhel-8-s390x-rpms \
-  --enable=ansible-2.8-for-rhel-8-s390x-rpms \
-  --enable=rhel-8-for-s390x-appstream-rpms
+2. Copy the `icic.crt` file to your certs directory
+```sh
+$ scp -r root@<CIC-control-node-ip>:/etc/pki/tls/certs/icic.crt /etc/pki/tls/certs/
 ```
 
-Then install the packages:
+3. Run source `icicrc` to set the environment variables.
 ```sh
-sudo yum install python3-openstackclient ansible python3-openstacksdk python3-netaddr jq
+$ source /opt/ibm/icic/icicrc
+ Please input the username: admin
+ Please input the password of admin:
 ```
 
-Make sure that `python` points to Python3:
+4. Verify the Red Hat machine is connected to ICIC successfully.
 ```sh
-sudo alternatives --set python /usr/bin/python3
+$ openstack project list
++----------------------------------+-------------+
+| ID                               | Name        |
++----------------------------------+-------------+
+| 9c301e5e7b3a4e48a45e21690357c1be | icicvm      |
+| 9deef79a109440c3b84e522296285904 | service     |
+| bb7cf9dc985046048ccd39059ebfd6bc | ibm-default |
++----------------------------------+-------------+
 ```
 
 ## OpenShift Configuration Directory
 
-All the configuration files, logs and installation state are kept in a single directory:
+This repository contains Ansible playbooks to deploy OpenShift on IBM® Cloud Infrastructure Center.
+
+### Download this playbook
 
+All the configuration files, logs and installation state are kept in a single directory:
 ```sh
 $ mkdir -p icic-upi
```

Comment:
`$ mkdir -p icic-upi` should be deleted.

Reply:
okay, got it.